README.md

```diff
@@ -1,499 +1,511 @@
-* SEMI-554 Added internal support for remote error.  Fix "cloverShouldHandleReceipts" and "disablePrinting" check to look for correct property. Use 1.2.0-rc1.0 of remote-pay-cloud-api.
-
-## Version 1.1.0
-## Version 1.1.0-rc6.4
-
-* SEMI-498 Revert update to use new schema objects.  New schema is slated for 1.2.
-
-## Version 1.1.0-rc6.3 (deprecated)
-
-* SEMI-498 Add ready checking before attempting remote calls.  Add request validation.  Inhibit multiple 'onReady' callbacks.  Update to use new schema objects.
-* SEMI-577 Add Declaration of support for Chrome version 54, Firefox version 49
-
-## Version 1.1.0-rc6.2
-
-* SEMI-541 Update remote pay cloud API classes to ver 1.1.0-rc5.1
-
-## Version 1.1.0-rc6.1
-
-* SVR-899 Handle reconnect requests from the server.
-
-## Version 1.1.0-rc6.0
-
-* PAY-1258 Fix documentation.  Set up flow to capture "REFUND_RESPONSE" and extract any additional failure info.  Fix namespace issues.  Change flow to depend on ACK messages (when supported).  Extend ping/pong timeout check.
-
-## Version 1.1.0-rc5.1
-
-* SEMI-493: Allow suppression of log messages. Log messages are now suppressed by default.  To enable default logging:
-```
-require("remote-pay-cloud").DebugConfig.loggingEnabled = true;
-```
-
-## Version 1.1.0-rc5.0
-
-* SEMI-438: Remove dependency on 'prototype.js'
-
-## Version 1.1.0-RC2
-
-* SEMI-457: Add remoteApplicationId to required configuration.
-* SEMI-434: Add ability to read card data.
-* SEMI-423: Added backwards compatibility For older versions of android remote-pay ACK messages.
-
-## Version 1.1.0-RC1
-
-A deprecated beta version of the Connector (Clover.js) is included in this version with `require` directive syntax, but will removed in the future.
-
-## Version [BETA](https://github.com/clover/remote-pay-cloud-BETA/tree/BETA_Final) 
-
-The beta version includes the earliest library as well as a server with examples of the functions. 
+![alt text](https://www.clover.com/assets/images/public-site/press/clover_primary_gray_rgb.png)
+
+# Clover SDK for JavaScript integration
+
+<!---
+!!NOTE!!  The following is automatically updated to reflect the npm version.
+See the package.json postversion script, which maps to scripts/postversion.sh
+Do not change this or the versioning may not reflect the npm version correctly.
+--->
+## Version
+
+Current version: 1.3.1-3
+
+## Overview
+This SDK provides an API that enables your JavaScript point-of-sale (POS) system to interface with a [Clover® payment device] (https://www.clover.com/pos-hardware/). Merchants can accept payments using credit, debit, EMV (chip card), and NFC contactless payments (including Apple Pay), as well as gift cards, EBT (electronic benefit transfer), and more. [Learn more about Clover® integrations](https://www.clover.com/integrations).

```

Comment:
It looks like you need to remove the space between the square brackets and the parentheses [] ()
[Clover® payment device] (https://www.clover.com/pos-hardware/)
for it to render correctly